packages/ember-routing-views/lib/components/link-to.js

```diff
@@ -334,7 +334,7 @@ linkToTemplate.meta.revision = 'Ember@VERSION_STRING_PLACEHOLDER';
   a supplied route by name.
 
   Instances of `LinkComponent` will most likely be created through
-  the `link-to` Handlebars helper, but properties of this class
```

Comment:
This seems weird? I don't think we use the phrase "component tag" to describe `{{link-to` anywhere else. I might just reword all this.

> `Ember.LinkComponent` components are invoked with `{{#link-to}}`. Properties
> of this class can be overridden with `reopen` to customize application-wide behavior.

Also change `@see` below this to `Ember.Templates.helpers.link-to`